### PR TITLE
More cluster lookup specs

### PR DESCRIPTION
**Pull request by xaviershay**
@eam @drcapulet @jeeyoungk  

Interesting addition is in the second commit.

### CODE REVIEW

File: spec/expand/simple/lookup.spec [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+%def:KEYS
 CLUSTER
 TYPE
 
-%{b}:KEYS
+%{def}:KEYS
+CLUSTER
+TYPE
+
+%{a}bc
```
Comment (eam) [inline spec/expand/simple/lookup.spec]: This is interpreted as %{ {a}bc } and should error since there's no %abc cluster

Comment (xaviershay) [inline spec/expand/simple/lookup.spec]: yeah there is, I renamed `%a` above

`spec/expand/simple/a.yaml → spec/expand/simple/abc.yaml`

**Comment by drcapulet** [comment]
:+1:

File: spec/expand/simple/lookup.spec (hunk 1 above)
Comment (eam) [inline spec/expand/simple/lookup.spec]: Aha, was looking at the clusters on master. LGTM.

**Pull request merged**